components/Updates.js

```diff
@@ -2,11 +2,32 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import { Button } from 'react-bootstrap';
 import { FormattedMessage } from 'react-intl';
+import styled from 'styled-components';
 
 import colors from '../lib/constants/colors';
 
+import Container from './Container';
 import StyledUpdate from './StyledUpdate';
 
+const LoadingContainer = styled.div`
+  color: ${colors.darkgray};
```

Comment:
`lib/constants/colors.js` is deprecated, let's take this as an opportunity to remove it from this file.

```suggestion
  color: #797D7F;
```

Reply:
Done. 👍🏽